releases/net472/KnownIssues/695400 - .NET 4.7.2 setup skips installing payload on Win10 machine when upgraded from Win7 with .NET 4.7.2 .md

```diff
@@ -0,0 +1,42 @@
+# .NET 4.7.2 setup fails to install files on OS when upgraded from Windows 7 SP1 or Windows Server 2008 R2 with .NET Framework 4.7.2 .
+
+## Symptoms
+When you install the .NET Framework 4.7.2 on Windows 10 version 1803 or lower and your system was upgraded from Windows 7 SP1 with .NET Framework 4.7.2 , .NET 4.7.2 setup installation reports success but fails to install .NET 4.7.2 files in the machine.
+
+Consider the following scenario:
+  1. On a Windows 7 SP1 or Windows Server 2008 R2 , install the .NET Framework 4.7.2
+  2. Upgrade the machine to Windows 10 Fall Creators (version 1709) or lower. Upgraded OS can be any version of Windows 8.1 or Windows Server 2008 R2.
```

Comment:
- Upgraded OS -- The upgraded OS
- Windows 8.1 or --> Windows 8.1, Windows 10, or